app/models/ChargeDetailsFilter.scala

```diff
@@ -16,6 +16,8 @@
 
 package models
 
+import play.api.mvc.PathBindable
+
 sealed trait ChargeDetailsFilter
 
 object ChargeDetailsFilter
```

Comment:
There are warnings on this file suggesting that the import below should be added:- import scala.language.implicitConversions